grakn-bootup/src/main/java/ai/grakn/bootup/AbstractProcessHandler.java

```diff
@@ -33,44 +33,34 @@
  */
 public abstract class AbstractProcessHandler {
 
-    public final static long WAIT_INTERVAL_S=2;
+    public static final long WAIT_INTERVAL_S=2;
+    public static final String BIN = "/bin/sh";
```

Comment:
maybe rename it to SHELL? BIN is a bit vague

Reply:
Good point 👍